README.md

```diff
@@ -90,19 +90,12 @@ sudo make install
  | path_angle_cost_weight    | double |                                                                                                             |
  | path_angle_cost_power     | int    |                                                                                                             |
 
-#### ReferenceTrajectoryCritic params
+#### PathAlignCritic params
```

Comment:
Add `retry_attempt_limit` to docs for controller level

Reply:
added